bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/WemoUtil.java

```diff
@@ -52,84 +47,6 @@ public static String substringBetween(@Nullable String string, String begin, Str
         return "";
     }
 
-    public static String unescape(final String text) {
-        StringBuilder result = new StringBuilder(text.length());
-        int i = 0;
-        int n = text.length();
-        while (i < n) {
-            char charAt = text.charAt(i);
-            if (charAt != '&') {
-                result.append(charAt);
-                i++;
-            } else {
-                if (text.startsWith("&amp;", i)) {
-                    result.append('&');
-                    i += 5;
-                } else if (text.startsWith("&apos;", i)) {
-                    result.append('\'');
-                    i += 6;
-                } else if (text.startsWith("&quot;", i)) {
-                    result.append('"');
-                    i += 6;
-                } else if (text.startsWith("&lt;", i)) {
-                    result.append('<');
-                    i += 4;
-                } else if (text.startsWith("&gt;", i)) {
-                    result.append('>');
-                    i += 4;
-                } else {
-                    i++;
-                }
-            }
-        }
-        return result.toString();
-    }
-
-    public static String unescapeXml(final String xml) {
-        Pattern xmlEntityRegex = Pattern.compile("&(#?)([^;]+);");
-        // Unfortunately, Matcher requires a StringBuffer instead of a StringBuilder
-        StringBuffer unescapedOutput = new StringBuffer(xml.length());
-
-        Matcher m = xmlEntityRegex.matcher(xml);
-        Map<String, String> builtinEntities = null;
-        String entity;
-        String hashmark;
-        String ent;
-        int code;
-        while (m.find()) {
-            ent = m.group(2);
-            hashmark = m.group(1);
-            if ((hashmark != null) && (hashmark.length() > 0)) {
-                code = Integer.parseInt(ent);
-                entity = Character.toString((char) code);
-            } else {
-                // must be a non-numerical entity
-                if (builtinEntities == null) {
-                    builtinEntities = buildBuiltinXMLEntityMap();
-                }
-                entity = builtinEntities.get(ent);
-                if (entity == null) {
-                    // not a known entity - ignore it
-                    entity = "&" + ent + ';';
-                }
-            }
-            m.appendReplacement(unescapedOutput, entity);
-        }
-        m.appendTail(unescapedOutput);
-
-        return unescapedOutput.toString();
-    }
```

Comment:
This method cannot be replaced by https://github.com/openhab/openhab-core/blob/14f8490d1cf67bcfce9d6b12e8997fb433317a03/bundles/org.openhab.core/src/main/java/org/openhab/core/util/StringUtils.java#L288-L310. Please check implementation.